Controllers/CustomerController.cs

```diff
@@ -32,20 +37,24 @@ public IActionResult Index()
         [HttpGet("details/{CustomerId}")]
         public string Details(int CustomerId)
         {
-            Console.WriteLine(CustomerId);
-            if (_db.Customer.Find(CustomerId) != null)
-            {
-                var objList = _db.Customer.Find(CustomerId);
-                // var objList2 = (IEnumerable<Customer>)_db.Customer.Where(b=>b.CustomerId==CustomerId);
+            try{
+                if (_db.Customer.Find(CustomerId) != null)
+                {
+                    var objList = _db.Customer.Find(CustomerId);
+                    // var objList2 = (IEnumerable<Customer>)_db.Customer.Where(b=>b.CustomerId==CustomerId);
 
                 string obj_json = JsonConvert.SerializeObject(objList);
 
-                return obj_json; // Ok - Library for returning Json response
+                    return obj_json; 
+                }
+                else
+                {
+                    return "Not Found";
+                    //return NotFound();
+                }
             }
-            else
-            {
-                return "Not Found";
-                //return NotFound();
+            catch(Exception e){
+                throw;
```

Comment:
I think just writing throw is going to stop the program without showing any error, so may be throwing a specific exception could be better for debugging purposes.

Reply:
Sure, I will look into it